Jenkinsfile.release

```diff
@@ -0,0 +1,79 @@
+#!groovy
+
+node {
+    properties([
+        parameters([
+            string(
+                defaultValue: '',
+                description: 'SHA of an existing container image. Usually what\'s currently on staging.', 
+                name: 'GIT_COMMIT',
+                trim: false
+            )
+        ])
+    ])
+
+    try {
+        // Checkout the proper revision into the workspace.
+        stage('checkout') {
+            checkout([
+                $class: 'GitSCM',
+                branches: [[name: params.GIT_COMMIT]],
+                extensions: [[$class: 'PruneStaleBranch']],
+                userRemoteConfigs: [[
+                    credentialsId: 'azaveaci',
+                    url: 'git@github.com:open-apparel-registry/open-apparel-registry.git'
+                ]]
+            ])
+        }
+
+        env.AWS_PROFILE = 'open-apparel-registry'
+        env.AWS_DEFAULT_REGION = 'eu-west-1'
+
+        env.OAR_SETTINGS_BUCKET = 'openapparelregistry-testing-config-eu-west-1'
+
+        // If we don't do this, the showmigrations stage will throw
+        // django.core.exceptions.ImproperlyConfigured
+        stage('setup') {
+            wrap([$class: 'AnsiColorBuildWrapper']) {
+                sh './scripts/bootstrap'
+            }
+        }
+
+        env.OAR_SETTINGS_BUCKET = 'openapparelregistry-production-config-eu-west-1'
+
+        // Plan and apply the current state of the production infrastructure
+        // as outlined by whatever SHA is passed through as a build parameter.
+        stage('infra') {
+            // Environment used in Rollbar deploy notifications.
+            // https://docs.rollbar.com/reference#post-deploy
+            env.OAR_DEPLOYMENT_ENVIRONMENT = 'production'
+
+            wrap([$class: 'AnsiColorBuildWrapper']) {
+                sh 'docker-compose -f docker-compose.ci.yml run --rm terraform ./scripts/infra plan'
+                withCredentials([[$class: 'StringBinding',
+                                credentialsId: 'OAR_ROLLBAR_ACCESS_TOKEN',
+                                variable: 'OAR_ROLLBAR_ACCESS_TOKEN']]) {
+                    sh 'docker-compose -f docker-compose.ci.yml run --rm terraform ./scripts/infra apply'
+                }
+            }
+        }
+
+        // Kick off an ECS task to display any outstanding migrations.
+        stage('showmigrations') {
+            wrap([$class: 'AnsiColorBuildWrapper']) {
+                sh './scripts/manage ecsmanage -e production showmigrations'
```

Comment:
Frustratingly, some of the logs for this task came back empty again.

Reply:
I wonder if this is happening because the management command output is being buffered. I think it would be valuable to try setting the `PYTHONUNBUFFERED` environment variable for the CLI task definition to minimize I/O buffering.

See: https://docs.python.org/3/using/cmdline.html#envvar-PYTHONUNBUFFERED

Reply:
This discussion supports using this on AWS:

https://github.com/awslabs/amazon-sagemaker-examples/issues/319

Reply:
Logs still don't appear to show up:

https://eu-west-1.console.aws.amazon.com/ecs/home?region=eu-west-1#/clusters/ecsProductionCluster/tasks/feefe694-ec8f-449c-83ec-b66c04e65b5f/details

Maybe I need to inject that environment variable in a different way?

Reply:
Hm. Now wondering if this is an issue with our Django logging configuration. I noticed in `settings.py` that we don't have `LOGGING` configured. We have `LOGLEVEL`, but I'm not seeing that setting used any where, nor referenced in the Django docs.

It might be worth configuring Django logging to see if that works. If not, opening an issue so that we can address it later. Something like the snippet below adds support for `stdout` logging with a configurable `DJANGO_LOG_LEVEL`:

```python
import os

LOGGING = {
    'version': 1,
    'disable_existing_loggers': False,
    'handlers': {
        'console': {
            'class': 'logging.StreamHandler',
        },
    },
    'loggers': {
        'django': {
            'handlers': ['console'],
            'level': os.getenv('DJANGO_LOG_LEVEL', 'INFO'),
        },
    },
}
```

Reply:
Tried to configure that as well, https://eu-west-1.console.aws.amazon.com/ecs/home?region=eu-west-1#/clusters/ecsProductionCluster/tasks/30d11c21-a9e8-44b5-8006-0d663db7e9b9/details, still doesn't work. 

I opened #358.